### ISSUE TITLE
Orphan Controller:MechanicalVentilation object

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#641]
I made a three loop model with 6 thermal zones. Loops A and B are single zone loops. Initially I hooked Zone 1 up to loop A, Zone 2 up to Loop B, and Zones 3-6 up to loop C.

I then disconnected Zone 3 from Loop C, disconnected Zone 1 from Loop A, and then connected Zone 3 to Loop A.

When I ran I got this error. Below are the three Controller:MechanicalVentilation objects. 

   ** Severe  ** IP: IDF line~7822 Object=CONTROLLER:MECHANICALVENTILATION, name=CONTROLLER MECHANICAL VENTILATION 1, Required Field=[Zone 1 Name] was blank.

@kbenne cc: @asparke2 I'll email the OSM file. This was from a clean file i made today with the 1.1.0 installer.

Controller:MechanicalVentilation,
  Controller Mechanical Ventilation 1,    !- Name
  Always On Discrete,                     !- Availability Schedule Name
  No,                                     !- Demand Controlled Ventilation
  ZoneSum,                                !- System Outdoor Air Method
  ;                                       !- Zone Maximum Outdoor Air Fraction {dimensionless}

Controller:MechanicalVentilation,
  Controller Mechanical Ventilation 3,    !- Name
  Always On Discrete,                     !- Availability Schedule Name
  No,                                     !- Demand Controlled Ventilation
  ZoneSum,                                !- System Outdoor Air Method
  ,                                       !- Zone Maximum Outdoor Air Fraction {dimensionless}
  Thermal Zone 1,                         !- Zone Name 1
  LargeHotel Banquet Outdoor Air,         !- Design Specification Outdoor Air Object Name 1
  Thermal Zone 1 Design Spec Zone Air Dist, !- Design Specification Zone Air Distribution Object Name 1
  Thermal Zone 4,                         !- Zone Name 2
  LargeHotel Cafe Outdoor Air,            !- Design Specification Outdoor Air Object Name 2
  Thermal Zone 4 Design Spec Zone Air Dist, !- Design Specification Zone Air Distribution Object Name 2
  Thermal Zone 5,                         !- Zone Name 3
  LargeHotel GuestRoom Outdoor Air,       !- Design Specification Outdoor Air Object Name 3
  Thermal Zone 5 Design Spec Zone Air Dist, !- Design Specification Zone Air Distribution Object Name 3
  Thermal Zone 6,                         !- Zone Name 4
  LargeHotel Corridor Outdoor Air,        !- Design Specification Outdoor Air Object Name 4
  Thermal Zone 6 Design Spec Zone Air Dist; !- Design Specification Zone Air Distribution Object Name 4

Controller:MechanicalVentilation,
  Controller Mechanical Ventilation 2,    !- Name
  Always On Discrete,                     !- Availability Schedule Name
  No,                                     !- Demand Controlled Ventilation
  ZoneSum,                                !- System Outdoor Air Method
  ,                                       !- Zone Maximum Outdoor Air Fraction {dimensionless}
  Thermal Zone 2,                         !- Zone Name 1
  LargeHotel Laundry Outdoor Air,         !- Design Specification Outdoor Air Object Name 1
  Thermal Zone 2 Design Spec Zone Air Dist; !- Design Specification Zone Air Distribution Object Name 1

**Comment by kbenne** [comment]
@DavidGoldwasser can you send the file again or put it in the openstudio-files repo?

**Comment by kbenne** [comment]
@DavidGoldwasser don't bother I followed your exact steps and was able to reproduce this.  Weird.  I'm a little surprised this has managed to lurk so long.  Stay tuned for a fix.

**Comment by kbenne** [comment]
@DavidGoldwasser Oh.  I think I know what it is.  When you remove all zones from an AirLoopHVAC system, the terminal is also removed.  Attached zones with no equipment (the terminal counts as equipment) do not get added to the mechanical ventilation controller.  

You might want the terminal to hang around after detaching the last zone, or you might prefer more explicit messaging of the root error, but lets file those as separate features.  The behavior of removing the last zone has wavered, this works tried and true, so I'm inclined to leave it, but I would entertain a feature request for change in behavior.

Bottom line. check your model.  I am almost certain you don't have a terminal on that Loop A.

**Comment by kbenne** [comment]
Going with [this](https://github.com/NREL/OpenStudio/pull/1488) instead, which adds the untranslated objects, errors and warnings at @macumber's request.

**Comment by DavidGoldwasser** [comment]
Makes sense, users needs to add the terminal back on their own if they have added, and then deleted a zone from an air loop, and then want to add a new one back.
